src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,39 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the unnecessary empty lines (lines 3-6) to comply with the checklist.